src/components/ContentHighlights/data/constants.js

```diff
@@ -1,6 +1,22 @@
-// eslint-disable-next-line import/no-extraneous-dependencies
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
[curious] Why is this ESLint rule disabled/necessary?

Reply:
I had attempting to bring in another dependency beyond faker but had since removed it. But it still requires a es-lint disable on the next line at the very list to allow faker dependency to be brought into the folder.

![Screen Shot 2022-12-19 at 12 47 57 PM](https://user-images.githubusercontent.com/82611798/208488142-5a206216-7d03-4078-8907-33b49aba87a3.png)